packages/@sanity/cli/src/commands/init/initCommand.ts

```diff
@@ -28,14 +28,15 @@ Options
   --package-manager <name> Specify which package manager to use [allowed: ${allowedPackageManagersString}]
   --auto-updates Enable/disable auto updates of studio versions (default: true)
   --overwrite-files Overwrite existing files (default: false)
+  --skip-mcp Skip AI editor integration (MCP) setup
```

Comment:
The convention we're trying to align on for flags is the `--no-` prefix, see for instance `--no-typescript`. 

```suggestion
  --no-mcp Skip AI editor integration (MCP) setup
```